tests/dialogBoxes.spec.ts

```diff
@@ -10,8 +10,9 @@ test.describe('Dialog boxes', () => {
         await page.getByRole('link', { name: 'Pet Types' }).click()
         await expect(page.getByRole('heading')).toHaveText('Pet Types')
         await page.getByRole('button', { name: "Add" }).click()
+        await expect(page.locator('h2').nth(1)).toHaveText('New Pet Type')
```

Comment:
Can you find a better locator, that will not use index of the element but instead will search the header in the section of the page, responsible for adding new pet type?